tests/test_lua_details.cc

```diff
@@ -22,10 +22,21 @@
 
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
+#include <limits>
 #include <gul14/catch.h>
```

Comment:
I'm not sure how we order this? But usually I assume alphabetical until an empty line.
Empty line to separate different 'classes' of includes.
L is not before G or C.

_Edit: Ah I see you drop the empty line below. I think that is not so good; it becomes un-understandable how the ordering works._
_I let my editor sort those lines, and there the empty lines also help._

Reply:
I sometimes omit the empty lines if there are only single headers per category, but if your editor works better with them, I'll insert them.

Reply:
Done.